cmd/immuadmin/immuadmin.go

```diff
@@ -45,8 +45,9 @@ Environment variables:
   IMMUADMIN_ADDRESS=127.0.0.1
   IMMUADMIN_PORT=3322
   IMMUADMIN_MTLS=true`,
+		SilenceUsage:      true,
```

Comment:
@mmeloni I suppose this is silencing the usage for all commands, not just for `service`? If so, maybe we can set it dynamically, in the `service` sub-command only?

Reply:
Removed option in service subcommand. The main one is persisted also in subocmmands